.github/workflows/build.yaml

```diff
@@ -81,16 +83,16 @@ jobs:
 
       - name: Install dependencies
```

Comment:
the 'setup ubuntu container' was (and probably is) a temporary step i had to add since github had removed support for ubuntu-18 runners. i do not think we need that step, but just keeping it for future when ubuntu 20 support might go away and we might need to readd the
```yaml
    container:
      image: ubuntu:18.04
```
directive for ubuntu-20 :|